app/services/stripe_service/api/stripe_api_wrapper.rb

```diff
@@ -264,6 +264,14 @@ def update_account(community:, account_id:, attrs:)
       end
     end
 
+    def update_account_capabilities(community:, account_id:)
+      with_stripe_payment_config(community) do |payment_settings|
+        account = Stripe::Account.retrieve(account_id)
+        account.requested_capabilities = ['card_payments', 'transfers']
+        account.save
```

Comment:
Will this result in an API call to Stripe, if the account already had all the requested capabilities? Or will the save be a no-op in that case? Since the update is a long operation (can take hours), it might be more efficient if saving is skipped when unnecessary, in case we interrupt and restart the update.